haskell/src/Lorentz/Contracts/StablecoinPath.hs

```diff
@@ -22,3 +23,7 @@ stablecoinFA1_2Path = "./test/resources/stablecoin.fa1.2.tz"
 -- | The path to the compiled metadata registry.
 metadataRegistryContractPath :: FilePath
 metadataRegistryContractPath = "./test/resources/metadata.tz"
+
+-- | The path to nix's sources.json file.
+sourcesJsonPath :: FilePath
+sourcesJsonPath = "./test/resources/sources.json"
```

Comment:
btw, why isn't it inherited directly from `../nix/sources.json`?

Reply:
Correct me if I'm wrong, but I think when you're building the haskell project using nix with `nix build -L -f. lib`, you can't access any files outside the `./haskell` folder.

Reply:
hmm, yes, you're right